Code/Source/Clock/SimulationClock.cpp

```diff
@@ -40,7 +40,7 @@ namespace ROS2
         if (!m_clockPublisher)
         {   //Lazy construct
             auto ros2Node = ROS2Interface::Get()->GetNode();
-
+            
```

Comment:
nitpick: extra whitespace has been added on blank line

Reply:
Removed this empty line altogether